tests/test_remote_inventory.py

```diff
@@ -0,0 +1,113 @@
+import os
+import sys
+import unittest
+import tempfile
+from shutil import rmtree, copytree
+from distutils.dir_util import copy_tree
+from kapitan.cached import reset_cache
+from kapitan.cli import main
+from kapitan.remoteinventory.fetch import (
+    fetch_git_source,
+    fetch_git_inventories,
+    fetch_http_source,
+    fetch_http_inventories,
+)
+from kapitan.dependency_manager.base import DEPENDENCY_OUTPUT_CONFIG
+
+
+class RemoteInventoryTest(unittest.TestCase):
+    def setUp(self):
+        os.chdir(
+            os.path.join(
+                os.path.abspath(os.path.dirname(__file__)), "test_remote_inventory", "environment_one"
+            )
+        )
+
+    def test_fetch_git_inventory(self):
+        temp_dir = tempfile.mkdtemp()
```

Comment:
These tests look good in general. Just one comment, add a pythondoc to all these test_ functions with just a sentence about what each test does. I know we can read it from code, but if you check other test_*.py files you'll see pydoc comments on all functions making code searching for relevant tests much much simpler for devs. :)